python/lsst/afw/cameraGeom/_camera.cc

```diff
@@ -22,11 +22,11 @@
  */
 
 #include "pybind11/pybind11.h"
-#include <lsst/utils/python.h>
+#include <lsst/cpputils/python.h>
```

Comment:
@mwittgen I'm guessing this line should be removed because it's repeated a couple of lines below but without the `<>`.